frontend/src/queries/nodes/WebVitals/definitions.ts

```diff
@@ -72,10 +72,9 @@ export const getMetric = (
         ?.data.slice(-1)[0]
 }
 
-export const getMetricBand = (
-    value: number | undefined,
-    threshold: WebVitalsThreshold
-): WebVitalsMetricBand | 'none' => {
+export const getMetricBand = (value: number | undefined, metric: WebVitalsMetric): WebVitalsMetricBand | 'none' => {
+    const threshold = WEB_VITALS_THRESHOLDS[metric]
```

Comment:
👍 to moving this lookup here (and below in `getThresholdColor`), I think it makes everything that calls it feel cleaner